web/src/features/weather-layers/wind-layer/WindLayer.tsx

```diff
@@ -29,7 +29,8 @@ const createWindy = async (canvas: HTMLCanvasElement, data: any, map: MapboxMap)
 export default function WindLayer({ map }: { map?: MapboxMap }) {
   const [isMapMoving] = useAtom(mapMovingAtom);
   const [windy, setWindy] = useState<Maybe<WindyType>>(null);
-  const { ref, node, width, height } = useReferenceWidthHeightObserver();
+  const reference = useRef<HTMLCanvasElement>(null);
+  const { height = 0, width = 0 } = useResizeObserver<HTMLCanvasElement>({ ref: reference });
```

Comment:
and here we set it once again? Why the inconsistency and why is the initial values needed?